dali/base/dadfs.hpp

```diff
@@ -494,6 +498,13 @@ class DistributedFilePropertyLock {
     {
         return file->queryAttributes();
     }
+    IPropertyTree &queryHistory()
+    {
+        IPropertyTree *temp = &file->queryHistory();
+        if (!temp)
+            temp = &file->createHistory(); // takes ownership
+        return *temp;
+    }
```

Comment:
I did this by two reasons:
1. The attributes handled same way, Somebody implemented it and it was/is good.
2. There is no direct way to access property root of the file. I thought to implement that and using to access History but that changes reveals the whole property tree and enable to (unintentionally) modify/destroy whole tree. With these two method the access restricted to the Attr and History and I think it is more safe way.